integration/tests/cook/test_multi_user.py

```diff
@@ -252,6 +253,64 @@ def test_rate_limit_while_creating_job(self):
             finally:
                 util.kill_jobs(self.cook_url,jobs_to_kill)
```

Comment:
Can you run through a code formatter? There should be a space after the comma

Reply:
Done